### ISSUE TITLE
feat: add a scoreboard

### ISSUE THREAD

**Issue by pedro-rs** [courselab/snaskii22#97]
It would be nice to have a display of how many fruits have been eaten (equivalent to displaying snake's size/points).

**Comment by pedro-rs** [comment]
I'll just be waiting for PR #47 to be resolved before starting to work on this. 
Progress indicates it will soon be merged.

**Comment by MetlHedd** [comment]
How do you plan to make it?

**Comment by rzimmerdev** [comment]
I'd say this is doable even without the completion of PR #47, maybe at least the placeholders

**Comment by rzimmerdev** [comment]
Any created variables could also be reusable nonetheless, if that somehow could be worrying

**Comment by rzimmerdev** [comment]
Can be closed!
